packages/prisma/src/prisma/schema/element.prisma

```diff
@@ -89,11 +88,16 @@ model ElementInstance {
   elementData Json // contains a copy of relevant element data
 
   /// [PrismaElementResults]
-  results Json // contains the collection of gathered results
+  results Json // contains the collection of gathered results by logged in participants
 
-  responses       QuestionResponse[]
-  detailResponses QuestionResponseDetail[]
-  feedbacks       ElementFeedback[]
+  // TODO: make required after filling field
+  /// [PrismaElementResults]
+  anonymousResults Json? // contains the collection of gathered results by anonymous participants
```

Comment:
> should we make it required and set the default to an empty object (if that is possible)?

Making the field required is planned after the corresponding migration has been successful. Since we initialize the results field with a default structure based on the element type, I would suggest we do the same with the anonymousResults. If we define a default value, however, it could happen that this is forgotten in future implementations or might break at some point without anyone noticing.

Reply:
Corresponding requirements are represented in task management